### PR TITLE
Use USD in place of POLY

### PR DESCRIPTION
**Pull request by adamdossa**
Use USD pricing instead of POLY for ticker registration, token generation and module costs.

All prices are now denominated in USD, and converted to POLY at the point in time the transaction is executed.

Possible concern is calculating the correct amount of POLY to approve / transfer ahead of the operation, but this could be mitigated by a more stable oracle.

### CODE REVIEW

File: contracts/SecurityTokenRegistry.sol [hunk 1]
```diff
@@ -234,9 +253,7 @@ contract SecurityTokenRegistry is EternalStorage, Proxy {
         require(_owner != address(0), "Owner should not be 0x");
         require(bytes(_ticker).length > 0 && bytes(_ticker).length <= 10, "Ticker length range (0,10]");
         // Attempt to charge the reg fee if it is > 0 POLY
-        uint256 tickerFee = getUintValue(TICKERREGFEE);
```
Comment (maxsam4) [inline contracts/SecurityTokenRegistry.sol:237]: `TICKERREGFEE` is now fees in USD, I suppose? All comments still say that it is the fee in Poly. Can you please update the comments if it's USD indeed.

**Review by maxsam4 — CHANGES_REQUESTED** [review]
Changes look good to me. Just some code comments need updating.

File: contracts/SecurityTokenRegistry.sol (hunk 1 above)
Comment (maxsam4) [reply contracts/SecurityTokenRegistry.sol:237]: Same goes for `STLAUNCHFEE` / `_stLaunchFee`

Comment (adamdossa) [reply contracts/SecurityTokenRegistry.sol:237]: Thanks - I went through the code and updated all the comments I could find.

**Review by maxsam4 — APPROVED** [review]

**Review by satyamakgec — APPROVED** [review]
I already reviewed the PR but forgot to Approved....

**Pull request merged**